libraries/rng256/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = [
   "Fabian Kaczmarczyck <kaczmarczyck@google.com>",
   "Guillaume Endignoux <guillaumee@google.com>",
   "Jean-Michel Picod <jmichel@google.com>",
+  "Julien Cretin <cretin@google.com>",
```

Comment:
Same here, I don't think I touched this library except for trivial things.

Reply:
The library is arguably trivial, but your call!